pkg/kubelet/kubelet.go

```diff
@@ -1672,6 +1672,9 @@ func (kl *Kubelet) rejectPod(pod *api.Pod, reason, message string) {
 // can be admitted, a brief single-word reason and a message explaining why
 // the pod cannot be admitted.
 func (kl *Kubelet) canAdmitPod(pods []*api.Pod, pod *api.Pod) (bool, string, string) {
+	if !kl.nodeManager.NodeIsSchedulable() && !isStaticPod(pod) {
```

Comment:
Ahh, the name is confused, and I should read that method in detail. We can live with that name for now.